maths/area.py

```diff
@@ -82,34 +78,61 @@ def area_square(side_length: float) -> float:
     return side_length ** 2
 
 
-def area_triangle(base: float, height: float) -> float:
+def area_isosceles_triangle(base: float, height: float) -> float:

```

Comment:
This is incorrect. The area of the triangle is actually `1/2 * base * height`, here height represents the distance between the base and the top point and not the distance of any sides.